style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+    color: blue;
+}
```

Comment:
- [ ] Kindly use the class name given to the `h1` tag in your html file as a selector instead of using `h1` element directly as a selector. It is for HTML and CSS best practice purpose.

Reply:
I have effected this change